stories/components/SINInput.stories.js

```diff
@@ -6,13 +6,14 @@ export default {
   argTypes: {},
 };
 
-const Template = (args, { argTypes }) => ({
-  props: Object.keys(argTypes),
+const Template = (args) => ({
   components: { SINInput },
-  template: '<SINInput v-bind="$props" />',
+  setup() { return { args }; },
+  template: '<SINInput v-bind="args" />',
 });
 
 export const Example = Template.bind({});
 Example.args = {
   label: 'My Label:',
+  placeholder: '111 111 111',  
 };
```

Comment:
Yes it does. I removed the placeholder arg override so that people don't accidentally think it comes with that placeholder by default, but they can still edit the arg in storybook and see what happens. (and all other args)